### DISCUSSION TITLE
完蛋了，粗大问题了

### DISCUSSION THREAD

**Discussion by Immortal-Sty** [hiroi-sora/Umi-OCR#429] [Q&A] [answered]
我以为命令行和 http 服务无关，然后做了以下设置：

- 开启后隐藏到托盘；
- 隐藏托盘图标；
- 关闭所有快捷键；
- 关闭 http 服务；

现在已经完全搞不起来了，悲。

**Comment by Immortal-Sty** [comment]
有人知道该咋搞不？

**Comment by hiroi-sora** [comment, marked as answer]
不慌，重置一下配置即可~

1. 如果软件还挂在后台，进任务管理器关掉它。或者重启一下电脑。
2. 打开软件目录，删掉配置文件：`Umi-OCR\UmiOCR-data\.settings`
3. 重新打开软件，所有配置都会重置。

**Reply by Immortal-Sty** [reply]
感谢大佬！！！
